docs/faq.md

```diff
@@ -176,6 +179,30 @@ Metacontroller will take care of [reconciling your desired state][reconciling].
 [watches]: /features/#declarative-watches
 [reconciling]: /features/#declarative-reconcilitation
 
+### Can I call external APIs from my hook?
+
+Yes. Your hook code can do whatever it wants as part of computing a response to
+a Metacontroller hook request, including calling external APIs.
+
+The main thing to be careful of is to avoid synchronously waiting for
```

Comment:
... to avoid _synchronous waits_ for ... ?

Reply:
Do you find the original wording confusing? As long as the intended meaning is clear, I'd rather keep "waiting" since it sounds more colloquial. Treating "waits" as a noun in this context feels more like technical jargon to me.

Reply:
I found the original wording to be slightly confusing, but I understand what the sentence means, so it is fine to keep the original wording. :)